### PR TITLE
Added separate CHANGELOGs for agent and backend

### PR DESCRIPTION
**Pull request by dusktreader**
#### What
* Gathered significant changes from git history and compiled into CHANGELOG.
* Separated changelog for agent and backend.

#### Why
The main CHANGELOG was basically empty.

####
Clickup Link: https://app.clickup.com/t/18022949/LM-123

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: backend/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Added support for poetry to manage dependencies
+* Added support for release to pypicloud
+* Added authorization sub-project for security on AWS Lambda
+* Vendorized flexlm
+* Added support for deployment via terraform to AWS Lambda
+* Backend:
+  * Added alembic support
+  * Added bookings endpoints
+  * Added FastAPI app for backend
+* Agent:
```
Comment (heitorPB) [inline backend/CHANGELOG.rst]:
```suggestion
* Backend:

  * Added alembic support
  * Added bookings endpoints
  * Added FastAPI app for backend
  
* Agent:

```

rst needs empty lines around nested list items.

**Review by heitorPB — CHANGES_REQUESTED** [review]
Missing empty lines after the section names and around subitems in lists

File: backend/CHANGELOG.rst (hunk 1 above)
Comment (dusktreader) [reply backend/CHANGELOG.rst]: Addressed in latest commit.

**Review by jamesbeedy — APPROVED** [review]
👍

File: agent/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+============
+ Change Log
+============
+
+This file keeps track of all notable changes to license-manager-agent
+
+Unreleased
+----------
+* Added RLM parser
+* Converted agent to a CLI application (from FastAPI with intenral scheduler)
```
Comment (julianaklulo) [inline agent/CHANGELOG.rst]:
```suggestion
* Converted agent to a CLI application (from FastAPI with internal scheduler)
```

File: agent/CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Added support for deployment via terraform to AWS Lambda
+* Backend:
+
+  * Added alembic support
+  * Added bookings endpoints
+  * Added FastAPI app for backend
+
+* Agent:
+
+  * Skip eplilog cleanup loop if there are no bookings
```
Comment (julianaklulo) [inline agent/CHANGELOG.rst]:
```suggestion
  * Skip epilog cleanup loop if there are no bookings
```

File: agent/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  * Moved support functions to cmd_utils
+  * Epilog updates token count to account for bookings
+  * Added PRODUCT_FEATURE_RX, ENCODING, and TOOL_TIMEOUT to settings
+  * Update prolog to only track licenses that match the expected format
+  * Added feature flags for "booked" and "product_feature"
+  * Extra accounting to add used slurm licenses to the total
+  * Added forced reconcilliation to the prolog
```
Comment (julianaklulo) [inline agent/CHANGELOG.rst]:
```suggestion
  * Added forced reconciliation to the prolog
```

File: backend/CHANGELOG.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+* Added support for deployment via terraform to AWS Lambda
+* Backend:
+
+  * Added alembic support
+  * Added bookings endpoints
+  * Added FastAPI app for backend
+
+* Agent:
+
+  * Skip eplilog cleanup loop if there are no bookings
```
Comment (julianaklulo) [inline backend/CHANGELOG.rst]:
```suggestion
  * Skip epilog cleanup loop if there are no bookings
```

File: backend/CHANGELOG.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  * Moved support functions to cmd_utils
+  * Epilog updates token count to account for bookings
+  * Added PRODUCT_FEATURE_RX, ENCODING, and TOOL_TIMEOUT to settings
+  * Update prolog to only track licenses that match the expected format
+  * Added feature flags for "booked" and "product_feature"
+  * Extra accounting to add used slurm licenses to the total
+  * Added forced reconcilliation to the prolog
```
Comment (julianaklulo) [inline backend/CHANGELOG.rst]:
```suggestion
  * Added forced reconciliation to the prolog
```

**Review by julianaklulo — CHANGES_REQUESTED** [review]
Spotted a few typos

File: agent/CHANGELOG.rst (hunk 2 above)
Comment (dusktreader) [reply agent/CHANGELOG.rst]: Will be updated in next commit.

File: agent/CHANGELOG.rst (hunk 3 above)
Comment (dusktreader) [reply agent/CHANGELOG.rst]: Will be updated in next commit.

File: agent/CHANGELOG.rst (hunk 4 above)
Comment (dusktreader) [reply agent/CHANGELOG.rst]: Will be updated in next commit.

File: backend/CHANGELOG.rst (hunk 5 above)
Comment (dusktreader) [reply backend/CHANGELOG.rst]: Will be updated in next commit.

File: backend/CHANGELOG.rst (hunk 6 above)
Comment (dusktreader) [reply backend/CHANGELOG.rst]: Will be updated in next commit.

**Review by julianaklulo — APPROVED** [review]

**Review by heitorPB — APPROVED** [review]

**Pull request merged**